server/user-update.js

```diff
@@ -63,6 +63,7 @@ module.exports = function handle({ message = {} }, { ship, hull }) {
     }
 
     if (errors && errors.length > 0) {
+      hull.post(`/connector/${ship.id}/notifications`, { status: "error", message: "Script error" });
```

Comment:
ok, I went again through the specs and I'm not sure if we want to have that notification here -> if the `notifications` won't debounce and throttle the emails yet we can easily flood the email inbox, @unity thoughts?